config/core/deployments/webhook.yaml

```
@@ -86,15 +86,23 @@ spec:
         - name: https-webhook
           containerPort: 8443
 
-        readinessProbe: &probe
+        readinessProbe:
           periodSeconds: 1
           httpGet:
             scheme: HTTPS
             port: 8443
             httpHeaders:
             - name: k-kubelet-probe
               value: "webhook"
-        livenessProbe: *probe
+        livenessProbe:
+          periodSeconds: 1
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            httpHeaders:
+            - name: k-kubelet-probe
+              value: "webhook"
+          failureThreshold: 5
```

Think the default failureThreshold is 3, I say let's double it for liveness (unless anyone feels strongly otherwise)

```suggestion
          failureThreshold: 6
```